.github/workflows/nodejs.yml

```
@@ -0,0 +1,128 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [8.x]
```

Don't we want something newer, e.g.?
```suggestion
        node-version: [10.x, 12.x, 14.x, 15.x]
```

---

i've removed 10.x as well, end of life is announced to 2021-04-30